### ISSUE TITLE
On multi-page form, file and address autocomplete fields get cleared when back button is clicked.

### ISSUE THREAD

**Issue by jwrudzin** [verbb/formie#293]
**Description**
On a multi-page form, when I use the "back" button, file and address autocomplete fields that were previously filled in are being cleared out.

**Steps to reproduce**
1. Create a multi-page form that uses server-side submit.
2. On an early page of the form, upload a file to a file field or add an address in an address autocomplete field.
3. Navigate to the next page, then navigate back to the page with the address autocomplete or file field.
4. The address autocomplete or file field will be cleared out.

### Form settings
Please pick from the options below.

- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.9
- Craft version: 3.5.17.1
- Multi-site: No

**Comment by engram-design** [comment]
The address persisting should be fixed in the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.9",
  "...": "..."
}
```

Then run `composer update`.

As for the file upload, that's a tricky one. From what I've read, once the file has been uploaded, we cannot assign a `value` to that field - it's simply ignored as an attribute. Notably, the file is certainly uploaded as soon as you hit next, and is stored against the incomplete submission. Going back and forth between pages won't effect the uploaded file - despite it not appearing - and you can overwrite the file with a new one.

For now though, I've added a summary below the field to at least show some files have been uploaded.

**Comment by engram-design** [comment]
Fixed in [1.3.10](https://github.com/verbb/formie/releases/tag/1.3.10)
